src/output_writers.jl

```diff
@@ -44,25 +48,34 @@ filename(fw::Checkpointer, name, iteration) = filename(fw, "model_checkpoint", i
 function write_output(model::Model, chk::Checkpointer)
     filepath = joinpath(chk.dir, filename(chk, model.clock.iteration))
 
-    # Do not include the spectral solver parameters. We want to avoid serializing
+    forcing_functions = model.forcing
+
+    # Do not include forcing functions and FFT plans. We want to avoid serializing
     # FFTW and CuFFT plans as serializing functions is not supported by JLD, and
     # seems like a tricky business in general.
+    model.forcing = nothing
     model.poisson_solver = nothing
 
+    println("WARNING: Forcing functions are not serialized!")
+
     println("[Checkpointer] Serializing model to disk: $filepath")
     f = JLD.jldopen(filepath, "w", compress=true)
     JLD.@write f model
     close(f)
 
-    # Reconstruct PoissonSolver struct with FFT plans ?
+    println("[Checkpointer] Reconstructing FFT plans...")
     metadata, grid, stepper_tmp = model.metadata, model.grid, model.stepper_tmp
-    if metadata.arch == :cpu
+    if metadata.arch == :CPU
```

Comment:
Why do we save FFT plans?

It makes sense that a user should plan new FFTs if they are going to run a model, even if they are restarting a checkpoint on the same computer.

@christophernhill what do you think?

Reply:
We don't. We set `model.poisson_solver = nothing` before checkpointing then reconstruct the FFT plans afterwards.

So you're saying the user should be manually planning the FFTs themselves?